test/extended/apiserver/health_endpoints.go

```diff
@@ -115,3 +164,15 @@ func testPath(transport http.RoundTripper, path string, requiredChecks sets.Set[
 	}
 	return nil
 }
+
+func checkFlatform(ctx context.Context, name string, oc *exutil.CLI) error {
```

Comment:
I would change the signature of this function to `isSupportedPlatform(ctx, name, oc) bool`.
For handling errors inside this function I would use `framework.ExpectNoError(err)`.